docs/security.rst

```diff
@@ -310,3 +310,20 @@ standard port 443, you'll need to configure that too. Be aware that super user p
     # Optionally, set the server to listen on the standard SSL port.
     web_server_port = 443
     base_url = http://<hostname or IP>:443
+
+Impersonation
+'''''''''''''
+
+Airflow has the ability to impersonate a unix user while running task
+instances based on the task's ``run_as_user`` parameter, which takes a user's name.
+
+*NOTE* For impersonations to work, Airflow must be run with `sudo` as subtasks are run
```

Comment:
Actually it looks like the current implementation requires passing relevant parts of the airflow.cfg to the impersonated user via chown so root access would be required by the airflow user after all.

This was the approach you and I discussed in https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Impersonation+Design+Doc

This sounds like something that we should discuss on Hangouts, will shoot you a message.